package.json

```diff
@@ -5,6 +5,7 @@
 	"scripts": {
 		"prepare": "husky install",
 		"dev": "concurrently \"vite dev\" typesafe-i18n",
+		"docker:dev": "concurrently \"vite dev --host 0.0.0.0\" typesafe-i18n",
```

Comment:
No need for an extra script here. The `node build` command will work automatically after the `build` directory is created during the build process.